package.json

```diff
@@ -29,7 +29,7 @@
   "license": "MIT",
   "dependencies": {
     "@conversationlearner/sdk": "0.255.0",
```

Comment:
I think this would be: `0.256.0` now to get the version of SDK which also has preview1.2